package.json

```diff
@@ -1,46 +1,90 @@
 {
-  "name": "express-tests",
-  "version": "1.0.0",
-  "main": "index.js",
+  "name": "game-catalog-starter",
+  "version": "1.0.3",
+  "main": "index.ts",
   "license": "MIT",
-  "dependencies": {
-    "body-parser": "^1.19.0",
-    "dotenv": "10.0.0",
-    "express": "^4.17.1",
-    "mongodb": "^4.2.0",
-    "node-fetch": "^2.6.1",
-    "nunjucks": "^3.2.3",
-    "slug": "^5.1.0",
-    "uuid": "^8.3.2"
-  },
   "scripts": {
     "build": "tsc",
     "dev": "nodemon",
-    "populate-db": "node src/scripts/populate-db.js",
-    "destroy-db": "node src/scripts/destroy-db.js",
+    "populate-db": "ts-node ./src/scripts/populate-db.ts",
+    "destroy-db": "ts-node ./src/scripts/destroy-db.ts",
     "format": "prettier --write .",
     "lint": "eslint \"{src,tests}/**/*.{js,ts,jsx,tsx}\"",
     "start": "node dist/index.js",
```

Comment:
It's not in the right folder anymore :) Script of folder architecture needs to be updated